parser/tests/Create.test

```diff
@@ -139,6 +139,62 @@ CREATE TABLE test (attr INT) CHECK(attr>0)
                              ^
 ==
 
+# Empty partitioning attribute is not allowed
```

Comment:
Thanks @jianqiao for pointing this out. This is exactly what I was looking for. Thanks again! :)